src/components/Trade/Components/TradeQuotes/TradeQuote.tsx

```diff
@@ -208,14 +208,7 @@ export const TradeQuoteLoaded: React.FC<TradeQuoteLoadedProps> = ({
           <RawText color='gray.500'>
             <FaGasPump />
           </RawText>
-          {
-            // We cannot infer gas fees for 1inch swapper before an amount is entered
```

Comment:
This doesn't hold true anymore now with the Oracle, correct?

Reply:
We would have been able to get gas fees regardless, just weren't. So not directly related, but it is using data from the gas oracle now as well